doc/examples/reconst_csd.py

```diff
@@ -70,8 +73,8 @@
 """
 0.21197
 
-We can double check that we have a good response function by visualizing the
-response's function's ODF. Here is how:
+We can double-check that we have a good response function by visualizing the
+response's function's ODF. Here is how you would do that:
```

Comment:
typo: response function's 
And I would remove all from Here ... 
I think it is redundant